### PR TITLE
Add documentation for installing data manager.

### PR DESCRIPTION
**Pull request by swgupta-vmw**
- Added the data manager details in a new supervisor-datamgr.md file
- Added the datamgr ova image link. The ova is uploaded to a aws S3 bucket
    https://vsphere-velero-datamgr.s3-us-west-1.amazonaws.com/datamgr-ob-17253392-photon-3-release-1.1.ova

TODO: What do we officially call supervisor and guest clusters

Signed-off-by: Swati Gupta <swgupta@vmware.com>

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/404/

File: docs/supervisor-datamgr.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Data manager configuration for vSphere with Kubernetes
+In a vSphere with Kubernetes environment, the data manager should be installed as a Virtual Machine on vSphere. It does not come
+up as container, as it does in a kubernetes vanilla vSphere environment. This is due to networking limitation in accessing
+vCenter/management network from TKGS clusters or PodVMs.
+
+This document discusses the installation procedure for backup data manager for velero plugin for vSphere with kubernetes.
+
+## Changes with release 1.1.0
+Support of vSphere with kubernetes (supervisor/workload cluster and TKGS cluster) is being added with the release 1.1.0 release.
```
Comment (xing-yang) [inline docs/supervisor-datamgr.md:9]: We should probably add a table to list all the terms.

File: docs/supervisor-datamgr.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Data manager configuration for vSphere with Kubernetes
+In a vSphere with Kubernetes environment, the data manager should be installed as a Virtual Machine on vSphere. It does not come
```
Comment (xing-yang) [inline docs/supervisor-datamgr.md:2]: Can you add a section explaining what does Data Manager do?  When it is required?

File: docs/supervisor-datamgr.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   - guestinfo.cnsdp.updateKubectl (default false, to avoid kubectl from wcp master on every VM restart)
+3. Power On the Data Manager VM
+
+## Debugging Data Manager issues, container image
+- Login to Data Manager VM
+  - Power on the VM and login - ssh or web console
+  - First time root password is "changeme". Update it.
+- Check velero data manager status and logs
+  - "docker ps" to confirm data manager container is running.
+  - "docker logs velero-datamgr" to get container logs
```
Comment (xing-yang) [inline docs/supervisor-datamgr.md:62]: docker command does not work any more on Supervisor Cluster?

File: docs/supervisor-datamgr.md (hunk 2 above)
Comment (swgupta-vmw) [reply docs/supervisor-datamgr.md:2]: Done.

File: docs/supervisor-datamgr.md (hunk 3 above)
Comment (swgupta-vmw) [reply docs/supervisor-datamgr.md:62]: This command is to be executed inside the data manager VM.

File: docs/supervisor-datamgr.md (hunk 1 above)
Comment (swgupta-vmw) [reply docs/supervisor-datamgr.md:9]: Would this be there in the main documentation? Lets add it once all the documentation has been added.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/416/

File: docs/supervisor-datamgr.md (hunk 3 above)
Comment (xing-yang) [reply docs/supervisor-datamgr.md:64]: Right.

File: docs/supervisor-datamgr.md (hunk 1 above)
Comment (xing-yang) [reply docs/supervisor-datamgr.md:13]: Sure.

**Review by xing-yang — APPROVED** [review]
LGTM

**Pull request merged**